### PR TITLE
create a module enumerable and class called list

### PR DESCRIPTION
**Pull request by EyasuTeshome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# enumerable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (GzimAsani) [inline README.md:1]: - [ ] Kindly update the pic since it is broken

File: enumerable.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  def filter(&block)
+    result = []
+    each do |element|
+      result << element if block.call(element) == true
+    end
+    result
+  end
+end
+Footer
```
Comment (GzimAsani) [inline enumerable.rb]: - [ ] I think this was added by mistake, kindly remove it

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+
+- GitHub: [@EyasuTeshome](https://github.com/EyasuTeshome)
+- Twitter: [@EyasuTeshome](https://twitter.com/EyasuTeshome)
+- LinkedIn: [Eyasu Teshome](https://linkedin.com/in/EyasuTeshome)
+
+👤 **Mehdi Rahal**
+
+- GitHub: [@Mehdi-Rh](https://github.com/Mehdi-Rh)
+- Twitter: [@MRahal92](https://twitter.com/MRahal92)
+- LinkedIn: [mehdi-rahal22](https://www.linkedin.com/in/mehdi-rahal22/)
```
Comment (GzimAsani) [inline README.md:26]: - [ ] ALl authors should be present

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello **TEAM**     :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your project it is working as expected  :+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @EyasuTeshome, @Carshy and @Mehdi-Rh 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**